src/Compilers/CSharp/Portable/Symbols/TypeUnification.cs

```diff
@@ -289,9 +289,7 @@ private static bool Contains(TypeSymbol type, TypeParameterSymbol typeParam)
                         NamedTypeSymbol namedType = (NamedTypeSymbol)type;
                         while ((object)namedType != null)
                         {
-                            var typeParts = namedType.IsTupleType ?
-                                namedType.TupleElementTypes.SelectAsArray(TypeMap.AsTypeSymbolWithAnnotations) :
-                                namedType.TypeArgumentsNoUseSiteDiagnostics;
+                            var typeParts = namedType.IsTupleType ? namedType.TupleElementTypes : namedType.TypeArgumentsNoUseSiteDiagnostics;
```

Comment:
Can you take a note to add tests on type unification? I don't think it was in scope for this PR.
`class C : I<T?> , I<string> { } // no warning?`
`class C : I<T?>, I<string?> { } // warning?`
`class C : I<T>, I<string?> { } // no warning?`
Also should be tested with `int` instead of `string`.